integrations/vite.md

```diff
@@ -237,6 +237,12 @@ export default {
 
 ### [Alias Config](https://windicss.org/posts/v30.html#alias-config)
```

Comment:
I find this habit (of linking elsewhere ON the section header) in the windi docs to be rather confusing.

Why is the only place that this is documented in the release docs?

I hope the below suggestion is acceptable.

Reply:
I would suggest we remove the link on the headline entirely. The added reminder more clearly points to additional resources.